functions/01_TotalActivation/RunTotalActivation.m

```diff
@@ -106,8 +106,12 @@
             if (param.use_pfm==1)
                 fprintf('Launching MyTemporal with Paradigm Free Mapping\n');
                 tmt = toc;
-                [temp, param] = MyTemporal_pfm(TC_IN, param);
+                [temp,Activity_inducing,innovation, param] = MyTemporal_pfm(TC_IN, param);
                 fprintf('PFM completed in %.5f\n', toc-tmt);
+                if (k==param.Nit)
```

Comment:
Now we put the matrices in param so they go to ``Run_TA.m`` and we don't need to change the number of outputs of ``RunTotalActivation.m`` and it can be use both for the pfm and conv temporal